locale/en_US/submission.xml

```diff
@@ -332,12 +332,11 @@
 	<message key="submission.layout.confirmDeleteSupplementaryFile">Are you sure you want to permanently delete this supplementary file?</message>
 	<message key="submission.layout.deleteGalleyStylesheet">Delete this style sheet.</message>
 	<message key="submission.layout.editGalley">Edit a Layout Galley</message>
-	<message key="submission.layout.galleyFileData">Galley File Information</message>
 	<message key="submission.layout.galleyFormat">Galley Format</message>
 	<message key="submission.layout.galleyHTMLData">HTML Galley Files</message>
 	<message key="submission.layout.galleyImages">Images</message>
-	<message key="submission.layout.galleyLabelInstructions">The galley label is used to identify the item's file format (e.g. HTML, PDF, etc.).</message>
-	<message key="submission.layout.galleyLabel">Label</message>
+	<message key="submission.layout.galleyLabel">Galley Label</message>
+	<message key="submission.layout.galleyLabelInstructions">Typically used to identify the file format (e.g. PDF, HTML, etc.).</message>
```

Comment:
I don't see any downside to merging this; other translations based on the older English will be serviceable, and newer translations may also be clearer. Translators won't be notified of the change, thus the workload for them isn't increased.